_networkconfig/2_domaincontrollers.md

```diff
@@ -4,18 +4,18 @@ title: Domain Controllers
 collection: networkconfig
 permalink: networkconfig/domaincontrollers/
 ---
+# How do I generate and install Certification Authority (CA) certificates for Domain Controllers?
 
-To use smartcards and PIV credentials for network authentication, all the Domain Controllers need to have Domain Controller Authentication certificates.
+These procedures are intended for network and system administrators, or other IT professionals, who are responsible for the day-to-day network operations of Federal Government agencies. As part of their roles, these professionals will be authorized by their agencies to manage various aspects of their networks, including generating and installing CA certificates for their Domain Controllers. Domain Controller certificates are required in order for the agencies' authorized users to use PIV card credentials for network authentication. 
```

Comment:
the entire concept of using PIV with Windows with domains is extremely challenging IMO and so for example it's not documented anywhere I know of that you need to issue DC's certificates.   That one sentence has great value in opening one's eyes to the level of complexity involved.